recipes/cp2k/a100/environments.yaml

```diff
@@ -0,0 +1,64 @@
+cp2k-nvgpu:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@11
+  unify: true
+  specs:
+  - cuda@11.8
+  - intel-oneapi-mkl +cluster
+  - fftw@3.3.10 +openmp
+  - libxc ~cuda
+  - sirius +fortran +scalapack
+  - dbcsr@develop +cuda cuda_arch=70 +shared +openmp ^intel-oneapi-mkl ^cray-mpich
+  - cosma@master +gpu_direct +scalapack +shared ^intel-oneapi-mkl ^cray-mpich
+  - cp2k@master +libxc +libint +spglib +cosma +sirius ~elpa build_system=cmake ^intel-oneapi-mkl ^cray-mpich
+  variants:
+    - +mpi
+    - +cuda
+    - cuda_arch=80
+    - build_type=Release
+  views:
+    dev:
+      exclude: ["cp2k"]
+    scalapack:
```

Comment:
Should we call this view `cp2k-scalapack`? From the user's perspective, that is what the environment contains.
The same goes for the ELPA version below.